### PR TITLE
install classnames package to tidy up conditional class code

### PR DESCRIPTION
**Pull request by emma-floored**
@Jinsoo94 @MagdalenaHenke I installed the [classnames package](https://www.npmjs.com/package/classnames) so maybe we can cut down on some of our conditional class code

### CODE REVIEW

**Comment by Jinsoo94** [comment]
ohhh i love it!

File: packages/blocks-react/components/dropdowns/DropdownMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const React = require('react');
+const PropTypes = require('prop-types');
+
+
+const DropdownMenu = (props) => {
```
Comment (Jinsoo94) [inline packages/blocks-react/components/dropdowns/DropdownMenu.jsx]: this one and DropdownTrigger seems like they don't belong in this PR. Is this spillover from another pr?

File: packages/blocks-react/components/dropdowns/DropdownMenu.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const React = require('react');
```
Comment (MagdalenaHenke) [inline packages/blocks-react/components/dropdowns/DropdownMenu.jsx]: @emma-floored - did you want this to be in this PR?

File: packages/blocks-react/components/dropdowns/DropdownMenu.jsx (hunk 1 above)
Comment (emma-floored) [reply packages/blocks-react/components/dropdowns/DropdownMenu.jsx]: oops thanks

File: packages/blocks-react/components/dropdowns/PopoverMenu.jsx [hunk 3]
```diff
@@ -8,27 +9,29 @@ const keyControlledMenu = require('../wrappers/keyControlledMenu.jsx');
...
-    menuClasses += ' blx-popover-is-center';
-  }
+  const classes = classnames('blx-popover-wrapper', props.className);
+  const triggerClasses = classnames('blx-popover-trigger', {
+    'blx-active': props.isOpen,
+    'blx-disabled': props.isDisabled
+  });
+  const menuClasses = classnames('blx-popover-menu', {
+    'blx-hidden': !props.isOpen,
+    'blx-popover-is-left': props.position === 'left',
```
Comment (MagdalenaHenke) [inline packages/blocks-react/components/dropdowns/PopoverMenu.jsx]: Could also be 
```javascript
  const menuClasses = classnames(`blx-popover-menu blx-popover-is-${props.position}`, {
    'blx-hidden': !props.isOpen,
  });
```

I like shorter but I think what it currently is is more readable, throwing it out there and you can pick

**Comment by MagdalenaHenke** [comment]
Other than that one take it or leave it comment, looooooove this 🌮 and ❤️

File: packages/blocks-react/components/dropdowns/PopoverMenu.jsx (hunk 3 above)
Comment (emma-floored) [reply packages/blocks-react/components/dropdowns/PopoverMenu.jsx]: I think I will leave as-is just for a slight bit of extra protection against nonsense `position`s getting passed in
